code/go/0chain.net/smartcontract/dbs/event/transaction.go

```diff
@@ -158,9 +157,17 @@ func (edb *EventDb) UpdateTransactionErrors() error {
 		return err
 	}
 
-	if dbTxn := db.Exec("INSERT INTO transaction_errors (transaction_output, output_hash, count) "+
-		"SELECT transaction_output, output_hash, count(*) as count FROM transactions WHERE status = ? and created_at > ? "+
-		"GROUP BY output_hash, transaction_output", 2, lastDayString); dbTxn.Error != nil {
+	if dbTxn := db.Exec("INSERT INTO transaction_errors (transaction_output, count) "+
+		"SELECT transaction_output, count(*) as count FROM transactions WHERE status = ? and created_at > ? and transaction_output not like '%:%'"+
+		"GROUP BY transaction_output", 2, lastDayString); dbTxn.Error != nil {
+
+		logging.Logger.Error("Error while inserting transactions in transaction error table", zap.Any("error", dbTxn.Error))
+		return dbTxn.Error
+	}
+
+	if dbTxn := db.Exec("INSERT INTO transaction_errors (transaction_output, count) "+
```

Comment:
this won't work well, process errors after you grouped them in golang

Reply:
Yes, making changes according to discussion.